src/box/lua/config/cluster_config.lua

```diff
@@ -191,22 +198,36 @@ local schema_name = 'nested_cluster_config'
 local nested_cluster_config = schema.new(schema_name, record_from_fields({
     instance_config_with_scope('global'),
     groups = schema.map({
-        key = name,
+        key = name_with_description('groups.*'),
```

Comment:
Should the description really be atteched to the key, not value?

Reply:
I think for both of them.

Reply:
Descriptions for map keys removed.